src/app/app.module.ts

```diff
@@ -94,10 +94,18 @@ import { TxConfirmNotificationProvider } from '../providers/tx-confirm-notificat
 import { TxFormatProvider } from '../providers/tx-format/tx-format';
 import { WalletProvider } from '../providers/wallet/wallet';
 
+/* Directives */
+
+import { CopyToClipboard } from '../directives/copy-to-clipboard/copy-to-clipboard';
+
 export function createTranslateLoader(http: HttpClient) {
   return new TranslatePoHttpLoader(http, 'assets/i18n', '.po');
 }
 
+let declarations: any = [
```

Comment:
Baybe this array should be name as "directives" ?

Reply:
Ok, I agree.

Reply:
Updated. Thanks!